logging/zerolog_adapter/logger.go

```diff
@@ -71,84 +72,24 @@ func getLogWriter(logFileName string) (io.Writer, error) {
 	return logFile, nil
 }
 
-func (l Logger) Debug(args ...interface{}) {
-	event := l.Logger.Debug()
-	if event == nil {
-		return
-	}
-	event.Timestamp().Msg(fmt.Sprint(args...))
-}
-
-func (l Logger) Debugf(format string, args ...interface{}) {
-	event := l.Logger.Debug()
-	if event == nil {
-		return
-	}
-	event.Timestamp().Msgf(format, args...)
-}
-
-func (l Logger) Info(args ...interface{}) {
-	event := l.Logger.Info()
-	if event == nil {
-		return
-	}
-	event.Timestamp().Msg(fmt.Sprint(args...))
-}
-
-func (l Logger) Infof(format string, args ...interface{}) {
-	event := l.Logger.Info()
-	if event == nil {
-		return
-	}
-	event.Timestamp().Msgf(format, args...)
+func (l Logger) Debugb() logging.EventBuilder {
```

Comment:
hmm, but why here?

and may be now letter **b** in func name doesn't need? 

and maybe Logger struct don't need anymore?

at least I think now we should make Logger field in Logger struct make private to avoid using not our components?

Reply:
- letter **b** will remove in another PR
- Logger and builder will help us to mock and test
- todo: make field private

Reply:
Made field private